test/pack/aws/cache.test.ts

```diff
@@ -88,24 +95,29 @@ describe("cache", () => {
             };
             assert.deepStrictEqual(c, e);
         });
+
     });
 
     describe("getCachePath", () => {
+
         it("should return a reasonable path", () => {
             const cc = {
                 bucket: "you-aint-goin-nowhere",
                 enabled: true,
                 path: "lazy/days",
+                region: "us-east-2",
             };
             const c = "i-am-a-pilgrim";
             const p = getCachePath(cc, c);
             const e = "lazy/days/i-am-a-pilgrim/cache.tar.gz";
             assert(p === e);
         });
+
     });
 
     describe("S3GoalCacheArchiveStore", () => {
-        before(function b(this: Mocha.Context): void {
+
+        before(function(this: Mocha.Context): void {
```

Comment:
tslint and prettier fight about this when the function is not named. Prettier puts a space after `function` and tslint complains about it.